src/providers/JoinStorageProvider.ts

```diff
@@ -13,4 +14,18 @@ export default interface JoinStorageProvider {
   // Set current database
   setDatabase(id: string);
 
+  // Set user
+  setUser(id: string);
+
+  // Connect to storage
+  connectToStorage(s: Store);
+
+  // Get storage address
+  getStorageAddress();
```

Comment:
Yeah you should probably strictly type all types here e.g
`getStorageAddress(void) : string;`
If you don't wanna it's fine we can do it later when we're properly documenting everything.

Reply:
added type signature